000-vip-init.php

```diff
@@ -245,6 +245,7 @@
 require_once __DIR__ . '/telemetry/tracks/class-tracks-client.php';
 require_once __DIR__ . '/telemetry/tracks/tracks-utils.php';
 require_once __DIR__ . '/telemetry/pendo/class-pendo.php';
+require_once __DIR__ . '/telemetry/pendo/class-pendo-javascript-library.php';
```

Comment:
This may trigger intermittent deploy error due to how deploy system works, needs to be guarded against with `file_exists` checks and if this class is a hard dependency somewhere that'd need to be wrapped as well. The checks can be removed after the initial deploy. Ping me in Slack if you need more details about it. @chriszarate